modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -53,9 +53,11 @@ storageConfig:
     skipTLS: false
 mirror:
   platform:
+    architectures: "multi"
```

Comment:
@snarayan-redhat did you confirm if this will work ? i mean i have always tried as below
architectures:
    - "multi"